src/current/cockroachcloud/migrations-page.md

```diff
@@ -75,7 +75,7 @@ The steps to convert your schema depend on your source dialect.
 ### Upload File
 <a name="upload-a-sql-dump"></a>
 
-The Schema Conversion Tool expects to analyze a SQL dump file containing only [data definition statements]({% link {{version_prefix}}sql-statements.md %}#data-definition-statements). 
+The MOLT Schema Conversion Tool expects to analyze a SQL dump file containing only [data definition statements]({% link {{version_prefix}}sql-statements.md %}#data-definition-statements). 
```

Comment:
Isn't this already specified in the heading of this document? Why do we need to specify `MOLT` every time? Is it just for consistency sake?

Reply:
@ajstorm mentioned that we need to brand it as MOLT, though I agree that prepending MOLT in every case feels a bit much. So I've dialed that back a bit on this page.